.github/workflows/tests_sources.yml

```diff
@@ -13,14 +11,26 @@ defaults:
     shell: bash
 
 jobs:
+  job-skipper:
+    runs-on: ubuntu-latest
+    outputs:
+      should_skip: ${{ steps.skip_check.outputs.should_skip }}
+    steps:
+      - id: skip_check
+        uses: fkirc/skip-duplicate-actions@master
+        with:
+          github_token: "${{ secrets.GITHUB_TOKEN }}"
+          paths_ignore: '["docs/**"]'
+
   cleanup-runs:
     runs-on: ubuntu-latest
     steps:
     - uses: rokroskar/workflow-run-cleanup-action@master
       env:
         GITHUB_TOKEN: "${{ secrets.GITHUB_TOKEN }}"
```

Comment:
Is this job still useful? Isn't ``job-skipper`` canceling previous workflows too?

Reply:
I am not sure about that, we should test it, but it is possible:
> Skip concurrent workflow-runs: If the same workflow is unnecessarily triggered twice, then one of the workflow-runs will be skipped. For example, this can happen when you push a tag right after pushing a commit.

Reply:
The 'Cancel outdated workflow-runs' feature might also allow to remove the cleanup job: https://github.com/fkirc/skip-duplicate-actions#cancel-outdated-workflow-runs